rdmd.d

```diff
@@ -437,70 +393,182 @@ private void unlockWorkPath()
     }
 }
 
-// Rebuild the executable fullExe starting from modules in myDeps
-// passing the compiler flags compilerFlags. Generates one large
-// object file.
+// Internal information needed for a build
+struct BuildInformation {
```

Comment:
Adding more structure has been looming over rdmd. When exactly the time has come depends on whom you ask. My guess is we can wait a bit more.